.circleci/config.yml

```diff
@@ -1,63 +1,165 @@
 version: 2
 jobs:
   build:
-    working_directory: ~/concur/developer.concur.com
+    working_directory: ~/test-repo
     docker:
-      - image: circleci/ruby:2.5.1
+      - image: cimg/ruby:2.7.3-node
+    environment:
+      BUNDLE_PATH: ~/repo/vendor/bundle 
     steps:
       - checkout
       - restore_cache:
           keys:
-          - devcenter-deps-{{ .Branch }}-{{ checksum "Gemfile" }}
-          - devcenter-deps-preview-      
-      - run: bundle install 
+            - rubygems-v1-{{ checksum "Gemfile.lock" }}
+            - rubygems-v1-fallback
+      - run: sudo apt-get install python3
+      - run: sudo apt-get update 
+      - run: sudo apt-get install -y python3-pip 
+      - run: sudo pip3 install pipenv 
+      - run: pipenv install
+      - run: pipenv install Unipath
+      - run:
+          name: Installing Ruby dependencies.
+          command: |
+                gem update --system
+                gem install bundler   
+      - run:
+          name: Bundle installation.
+          command: bundle check || bundle install
       - save_cache:
-          key: devcenter-deps-{{ .Branch }}-{{ checksum "Gemfile" }}
+          key: rubygems-v1-{{ checksum "Gemfile.lock" }}
           paths:
-          - ~/.bundle      
-      - run: ./build/write_branch.sh
-      - run: ./build/write_commit.sh
-      - run: ./build/write_forms.sh
-      - run: bundle exec jekyll build
+            - vendor/bundle
+      - run:
+          name: Widdershins Install and Execution.
+          command: |
+                sudo npm install -g widdershins
+                cd ./src/api-explorer/ && find . -name \*.json -type f -exec widdershins \{\} -o \{\}.md --omitHeader \;
```

Comment:
We don't need this here... we're doing it at 67 below

Reply:
Resolved.